src/components/ConfirmDialog/useConfirmationDialog.tsx

```diff
@@ -7,18 +7,8 @@
 
 import React, { useCallback } from 'react'
 import { Intent, Dialog, IDialogProps } from '@blueprintjs/core'
-import {
-  useModalHook,
-  Button,
-  ButtonProps,
-  Layout,
-  Container,
-  Icon,
-  Text,
-  ButtonVariation,
-  FontVariation,
-  Color
-} from '../../'
+import { useModalHook } from '@harness/use-modal'
```

Comment:
since UICore will itself import `use-modal`, and NGUI will also import it, we should  make sure it;s not bundled twice.

Reply:
webpack externals already takes care of it